gobreaker.go

```diff
@@ -9,15 +9,29 @@ import (
 )
 
 // State is a type that represents a state of CircuitBreaker.
-type State int
+type State uint8
 
 // These constants are states of CircuitBreaker.
 const (
-	StateClosed   State = iota
-	StateHalfOpen State = iota
-	StateOpen     State = iota
+	StateClosed State = iota
+	StateHalfOpen
+	StateOpen
 )
 
+// String implements stringer interface
```

Comment:
Minor, but this sentence is missing a period at the end. Which is inconsistent with all other comments here.